superbit_lensing/medsmaker/superbit/maker.py

```diff
@@ -0,0 +1,1406 @@
+"""
```

Comment:
Is this the modified version of MEDS that you were using before switching to the diff extender? We do not want this file in the repo

Reply:
Makes sense, we should definitely remove it now.